x-pack/plugins/ml/server/models/data_recognizer/modules/auditbeat_process_docker_ecs/ml/docker_high_count_process_events_ecs.json

```diff
@@ -0,0 +1,41 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Auditbeat: Detect unusual increases in process execution rates in docker containers (ECS)",
+  "groups": ["auditd"],
+  "analysis_config": {
+    "bucket_span": "10m",
+    "detectors": [
+      {
+        "detector_description": "high_non_zero_count partition container.name",
```

Comment:
I know this is the same as the current non ECS module, but I wonder if we should give this detector a more user-friendly description? `High process execution rate` ?